### PR TITLE
Add the feature page for Normalizing scheduling policy weights

### PR DESCRIPTION
**Pull request by MarSik**
Changes proposed in this pull request:

- A feature page for scheduling weight normalization

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @MarSik

This pull request needs review by: @doron-fediuck @rmohr @jtokar @yanirq

### CODE REVIEW

File: source/develop/release-management/features/sla/scheduling-weight-normalization.html.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+feature_name: Normalizing scheduling policy weights
+feature_modules: engine
+feature_status: Draft
+---
+
+## Normalizing scheduling policy weights
+
+### Summary
+
+The currently used weight policy units do not use any common result value range. Each unit reports numbers as needed and this causes issues with user configured preferences (policy unit multiplier), because for example memory (very high numbers) always wins over cpu (0-100).
```
Comment (mykaul) [inline source/develop/release-management/features/sla/scheduling-weight-normalization.html.md]: Everywhere, consider cpu -> CPU

File: source/develop/release-management/features/sla/scheduling-weight-normalization.html.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Benefit to oVirt
+
+The multiplier factor in the Scheduling policy edit dialog will become predictable. This will also allow the user to fine tune his preferences (for example: cpu vs. memory vs. weak affinity rules).
+
+### Implementation details
+
+#### Scheduling process changes
+
+The existing policy units won't be touched. The normalization process will be done by adding an extra step to the current Filter / Weight sequence. It will become Filter / Weight / Select.
```
Comment (mykaul) [inline source/develop/release-management/features/sla/scheduling-weight-normalization.html.md]: touched -> affected or changed.

File: source/develop/release-management/features/sla/scheduling-weight-normalization.html.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+### Dependencies / Related Features
+
+This normalization is needed as a ground work for all weak affinity features, including host affinity and Rack based balancing.
+
+### Documentation / External references
+
+- [Bug: Soft negative affinity](https://bugzilla.redhat.com/show_bug.cgi?id=1207255)
+- [Bug: Normalize policy unit weights](https://bugzilla.redhat.com/show_bug.cgi?id=1306263)
+
+### Testing
```
Comment (mykaul) [inline source/develop/release-management/features/sla/scheduling-weight-normalization.html.md]: Some notes on how one, via logging, can see the internals?

Comment (doron-fediuck) [inline source/develop/release-management/features/sla/scheduling-weight-normalization.html.md]: The following was added to the testing sections:
"The selection step result is visible in the DEBUG log (logger name: org.ovirt.engine.core.bll.scheduling.policyunits.RankSelectorPolicyUnit) in a CSV table form" etc.

**Pull request merged**